django/core/serializers/python.py

```diff
@@ -57,7 +81,16 @@ def handle_fk_field(self, obj, field):
         ):
             related = getattr(obj, field.name)
             if related:
-                value = related.natural_key()
+                natural_key_value = related.natural_key()
+
+                is_opt_out = (natural_key_value is None) or (
+                    natural_key_value == (related.pk,)
+                )
```

Comment:
This isn't identical with the other check, where non-tuples or 0-tuples are handled.